roles/openshift_cfme/tasks/storage/nfs.yml

```diff
@@ -0,0 +1,78 @@
+---
+# Tasks to statically provision NFS volumes
+# Include if not using dynamic volume provisioning
+
+- name: Note Storage Type - NFS
+  debug:
+    msg: Setting up NFS storage, openshift_cfme_storage_class is 'nfs'
+
+# TODO: I was going to try to apply the openshift_storage_nfs role to
+# handle this, however, that role is not written to be used by
+# itself. Attempting to use it to create CFME exports would just add
+# more hard-coded values to the role. That said, we're doing this here
+# manually for now until some one comes up with a better solution, or
+# the role is made to accept parameters in a more functional way.
+#
+# I can't really even include the openshift_storage_nfs role in here
+# to do basic setup stuff because it would just result in a lot of
+# unwanted exports getting set up for the users.
+
+- name: Ensure the base NFS exports directory exists
```

Comment:
Is there anything selinux specific that will need to happen?

Reply:
I am uncertain of that. However, on my working system here are the set contexts

```
[root@m01 ~]# ls -ldZ /exports/
drwxr-xr-x. root root system_u:object_r:unlabeled_t:s0 /exports/
```